electron-app/src/services/types-common.ts

```diff
@@ -1 +1,25 @@
 export type DSRMovementType = 'withdrawal' | 'deposit';
+
+export enum Location {
+  EXTERNAL = 'external',
+  KRAKEN = 'kraken',
+  POLONIEX = 'poloniex',
+  BITTREX = 'bittrex',
+  BINANCE = 'binance',
+  BITMEX = 'bitmex',
+  COINBASE = 'coinbase',
+  TOTAL = 'total',
```

Comment:
Can you remove this option (`TOTAL`)? It makes no sense in the context of adding locations. It's only used by the backend to show the total balances.